android/app/build.gradle

```diff
@@ -191,12 +191,10 @@ android {
 }
 
 dependencies {
-    implementation project(':react-native-webview')
```

Comment:
is there a reason you remove this?

Reply:
I did not remove it. There was some error and I run unlink... from the terminal. Did not know that it affects our project. sorry

Reply:
best practice is to do what you did

but since the upstream project does not change the file, we should keep it as close as possible